controls/tabview/styling/content-styling.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Content Styling
+page_title: .NET MAUI TabView Documentation - Content Styling
+description: Learn how to style the Telerik UI for .NET MAUI TabView Content by setting border thickness, color and corner radius.
+slug: tabview-content-styling
+tags: tabview, content, style, maui, dotnet
+position: 4
+---
+
+# .NET MAUI TabView Content Styling
+
+TabView's `ContentStyle` property allows you to style the content area. `TabViewContent` provides the following properties:
+
+* `BackgroundColor` (`Color`)&mdash;Specifies the background color of the content.
+* `BorderColor` (`Color`)&mdash;Specifies the border color of the content.
+* `BorderThickness` (`Thickness`)&mdash;Specifies the border thickness of the content.
+* `CornerRadius` (`Thickness`)&mdash;Specifies the corner radius of the content.
+* `ContentPadding` (`Thickness`)&mdash;Specifies the padding of the inner content of the content.
+
+<snippet id='tabview-styling-contentstyle' />
```

Comment:
Introduce the example, see the comment  in `controls/tabview/styling/header-styling.md`.